src/python/pants/backend/jvm/subsystems/jvm_platform.py

```diff
@@ -54,6 +54,8 @@ def register_options(cls, register):
              help='Compile settings that can be referred to by name in jvm_targets.')
     register('--default-platform', advanced=True, type=str, default=None, fingerprint=True,
              help='Name of the default platform to use if none are specified.')
```

Comment:
Would be good to expand the help here to better differentiate it from the runtime platform.

Reply:
I'll take a crack at it.